internal/interpreters/r_test.go

```diff
@@ -68,7 +68,7 @@ func (s *RSuite) TestInit() {
 
 	executor := executortest.NewMockExecutor()
 	executor.On("RunCommand", mock.Anything, []string{"--version"}, mock.Anything, mock.Anything).Return([]byte("R version 4.3.0 (2023-04-21)"), nil, nil)
-	executor.On("RunCommand", mock.Anything, []string{"-s", "-e", "renv::paths$lockfile()"}, mock.Anything, mock.Anything).Return([]byte(`[1] "/test/sample-content/shinyapp/renv.lock"`), nil, nil).Once()
+	executor.On("RunScript", mock.Anything, []string{"-s"}, "renv::paths$lockfile()", mock.Anything, mock.Anything).Return([]byte(`[1] "/test/sample-content/shinyapp/renv.lock"`), nil, nil).Once()
```

Comment:
If I'm understanding correctly we are changing these from `RunCommand` to `RunScript` because `R... -f` can be used. For my own edification what is the distinction for when to run using `RunScript` and `RunCommand`?

Reply:
A comment explaining the difference would be fantastic 👍